controller/Cargo.toml

```diff
@@ -16,7 +16,6 @@ kube-runtime = "0.75"
 lazy_static = "1"
 log = "0.4"
 model = { version = "0.0.2", path = "../model" }
-parse_duration = "2.1"
```

Comment:
Shouldn't this also be removed from the `Cargo.lock`?

Reply:
This is why the builds are failing. The Cargo.lock needs to be updated and checked in to the commit.

Reply:
Yeah, I had that ready but I didn't want to do a bunch of force-pushes. It'll be in the next push.